cdc/server.go

```diff
@@ -367,3 +357,25 @@ func findBestDataDir(candidates []string) (result string, ok bool) {
 
 	return result, ok
 }
+
+func checkAndWaitClusterVersion(
+	ctx context.Context, pdClient pd.Client, pdEndpoints []string, security *config.SecurityConfig,
+) {
+	const backoffOneSecond = 1000
+	const errorTiKVIncompatible = true
+
+	retry.Do(ctx, func() error {
+		var err error
+		for _, pdEndpoint := range pdEndpoints {
+			err = version.CheckClusterVersion(
+				ctx, pdClient, pdEndpoint, security, errorTiKVIncompatible)
```

Comment:
Seems the parameter `errorTiKVIncompatible` can be removed from function `CheckClusterVersion`